launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
Forget my original comment on this. It doesn't apply to this line anymore.

Anyway I don't think we need `expected_output` if we already store `expected_lines`. Since we don't support multi-line regex matching.